install/kubernetes/templates/operator_configmap.yaml

```
@@ -7,3 +7,4 @@ metadata:
   {{- include "tetragon-operator.labels" . | nindent 4 }}
 data:
   skip-crd-creation: {{ .Values.tetragonOperator.skipCRDCreation | quote }}
+  skip-tetragon-pod-crd: {{ .Values.tetragonOperator.skipTetragonPodCRD | quote }}
```

```suggestion
  skip-tetragon-pod-crd: {{  not .Values.tetragonPod.enabled | quote }}
```
let's just use `tetragonPod.enabled` to decide whether to create the CRD.

---

Sure, makes sense